app/models/post.rb

```diff
@@ -0,0 +1,19 @@
+class Post < ApplicationRecord
+  belongs_to :user
+  has_many :comments
+  has_many :likes
+
+  after_create :increment_posts_counter
+  before_destroy :decrement_posts_counter
+  def increment_posts_counter
+    user.increment!(:posts_counter)
+  end
```

Comment:
- [ ] Kindly make the `increment_posts_counter` method and the `decrement_posts_counter` private. Notice that the `recent_comments` method is above the private keyword.

e.g
```
def recent_comments
    comments.order(created_at: :desc).limit(5)
  end


private

def increment_posts_counter
    user.increment!(:posts_counter)
  end

 def decrement_posts_counter
    user.decrement!(:posts_counter)
  end
```
## Note that this applies to the likes and comment model